src/app/accounts/_forms/register.form.ts

```diff
@@ -6,6 +6,9 @@ export class RegisterForm {
         const builder = new FormBuilder()
         return builder.group({
             email: new FormControl(null, [Validators.required, Validators.email]),
+            first_name: new FormControl(null, [Validators.required]),
+            last_name: new FormControl(null, [Validators.required]),
+            nickname: new FormControl(null, [Validators.required, Validators.minLength(1)]),
```

Comment:
minLength(1) does nothing when the field is required, right?

Reply:
Oh I just did not want the nickname to be an empty space, so maybe I misunderstood how this could be done.

Reply:
Yeah, you are right, this was not required